### ISSUE TITLE
Investigate daily accessibility scan disk space issues

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#43418]
For the last week, the daily accessibility scan has been failing due to a lack of disk space on self-hosted GHA runners. This task is to investigate a problem and implement the fix. Coordination with the Infrastructure team and/or Release Tools team will likely be needed here.
